source/table-filters/apply-table-filters.txt

```diff
@@ -1,3 +1,46 @@
 ===================
 Apply Table Filters
-===================
\ No newline at end of file
+===================
+
+
+You can use table filters to migrate specific subsets of your table data.
+Table filters allow you to apply SQL where and row limit clauses
+when pulling data from your table.
+
+Before You Begin
+----------------
+
+Table filters are applied while you are creating mapping rules. See 
+:ref:`<rm-create-mapping-rules>` for more details.
+
+You can apply one filter per table. Each filter can have a SQL where 
+clause, a row limit clause, or both.
+
+Steps
+-----
+
+
+#. From the :guilabel:`Schema model` pane, click the table that you want to apply a filter to.
```

Comment:
(typo)
Edit Mapping Rule should probably be a guilabel.

Reply:
Yes agreed, updated.